agent/src/main/java/com/newrelic/agent/android/NewRelic.java

```diff
@@ -281,8 +285,14 @@ public void start(Context context) {
             return;
         }
         try {
-            AgentLogManager.setAgentLog(loggingEnabled ? new AndroidAgentLog() : new NullAgentLog());
-            log.setLevel(logLevel);
+            AgentLog logger = loggingEnabled ? new AndroidAgentLog() : new NullAgentLog();
```

Comment:
Oops, please disregard this. It was a _what if_ and should not have been included. I'll remove it in another commit.